docs/src/main/paradox/google-cloud-pub-sub-grpc.md

```diff
@@ -70,7 +70,7 @@ reference.conf
 Test Configuration
 : @@snip (/google-cloud-pub-sub-grpc/src/test/resources/application.conf)
 
-For more configuration details consider the underlying configuration for @extref:[Akka gRPC](akka-grpc:/client/configuration.html).
+For more configuration details consider the underlying configuration for @extref:[Apache Pekko gRPC](akka-grpc:/client/configuration.html).
 
 A manually initialized @scala[@scaladoc[GrpcPublisher](akka.stream.alpakka.googlecloud.pubsub.grpc.scaladsl.GrpcPublisher)]@java[@scaladoc[GrpcPublisher](akka.stream.alpakka.googlecloud.pubsub.grpc.javadsl.GrpcPublisher)] or @scala[@scaladoc[GrpcSubscriber](akka.stream.alpakka.googlecloud.pubsub.grpc.scaladsl.GrpcSubscriber)]@java[@scaladoc[GrpcSubscriber](akka.stream.alpakka.googlecloud.pubsub.grpc.javadsl.GrpcSubscriber)] can be used by providing it as an attribute to the stream:
```

Comment:
multiple references to akka namespaces here as well

Reply:
we haven't published the javadoc/scaladoc for pekko-connectors yet, so I don't want to link to vapourware - we can update these links when we have the javadoc/scaladoc for pekko-connectors published - I'd prefer to wait till we have changed the package names in the pekko-connectors to do this